scripts/babel_update.sh

```diff
@@ -54,4 +54,17 @@ pybabel update \
   -d superset/translations \
   --ignore-obsolete
 
+# Chop off last blankline from po/pot files, see https://github.com/python-babel/babel/issues/799
```

Comment:
Was the extra line causing a problem (e.g. linting issues) or is this just cleanup to feel good about things? Only asking because the other solution might be to tweak the linting rule for these files. This seems fine, I'm just curious. ¯\\\_(ツ)_/¯

Reply:
The duplicated empty line was reported by a linting rule, indeed.